Containerfile.dev

```diff
@@ -1,5 +1,5 @@
 ARG FEDORA_VERSION
-FROM fedora:${FEDORA_VERSION}
+FROM docker.io/library/fedora:${FEDORA_VERSION}
```

Comment:
Why not `registry.fedoraproject.org` then? :)

Reply:
I thought maybe dockerhub has the fastest support. But sure we can change it to that.

Reply:
What about for postgresql and rabbitmq. Should I change their registry aswell?

Reply:
I'm not sure if postgresql or rabbitmq could be found on `registry.fedoraproject.org`

Reply:
They dont exists under fedora registry. I thought maybe we can use redhat's registry or something.